gi_loadouts/face/wind/fclt.py

```diff
@@ -139,14 +139,16 @@ def weap_save(self) -> None:
                     refn=self.weap_area_refn.currentText(),
                 )
 
-                file.save(
+                save_status = file.save(
```

Comment:
Try to use similar naming conventions as used in other parts of the codebase (https://github.com/TAKI12T/gi-loadouts/blob/11337829e46ed61245fae9b0be8dcc4b32df47dc/gi_loadouts/face/wind/fclt.py#L168).

```suggestion
                status = file.save(
```